recipes/pyNSID/meta.yaml

```diff
@@ -0,0 +1,49 @@
+{% set name = "pyNSID" %}
+{% set version = "0.0.3" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: ed3afeba60af788def69e5facf8a0948be81ac0b92ba3d450e9aa7b691182b01
+
+build:
+  noarch: python
+  number: 0
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  host:
+    - python >=3
+    - pip
+    - pytest-runner
+  run:
+    - python >=3
+    - cytoolz
+    - dask-core >=0.10
+    - h5py >=2.6.0
+    - numpy >=1.10
+    - sidpy >=0.0.2
+    - six
+    - toolz
+
+test:
+  imports:
+    - pyNSID
+    
+about:
+  home: https://github.com/pycroscopy/pyNSID
+  license: MIT
+  license_family: MIT
+  license_file: {{ environ["RECIPE_DIR"] }}/LICENSE
```

Comment:
```suggestion
  license_file: LICENSE
```

Reply:
This is Gerd (@gduscher), and yes, I will be a maintainer of this package at conda-forge

Reply:
This is Rama (@ramav87), and yes, I will be a maintainer of this package at conda-forge